data/json/items/tool/landscaping.json

```diff
@@ -359,7 +359,7 @@
     "techniques": [ "WBLOCK_1", "BRUTAL", "WIDE", "SWEEP" ],
     "faults": [ { "fault_group": "handle_long", "weight_mult": 4 }, { "fault_group": "blade_general" } ],
     "flags": [ "DURABLE_MELEE", "REACH_ATTACK", "NONCONDUCTIVE", "SHEATH_SPEAR", "ALWAYS_TWOHAND" ],
-    "weapon_category": [ "SPEAR" ],
+    "weapon_category": [ "POLEARMS" ],
```

Comment:
`POLEARMS`? but it is like 150cm in length. a wooden spear is like 180cm and considered `SPEARS`.